targets/TARGET_STM/stm_spi_api.c

```diff
@@ -777,75 +800,225 @@ static inline int datasize_to_transfer_bitshift(uint32_t DataSize)
     }
 }
 
-int spi_master_write(spi_t *obj, int value)
+/**
+ * Check if SPI is busy in master mode.
+ */
+static inline int msp_busy(spi_t *obj)
```

Comment:
what does msp mean ?

Reply:
The are some slave SPI mode related functions with `ssp_` prefix (`ssp_readable`, `ssp_writable`, etc.). Probably it's abbreviation of slave SPI peripheral (or something like that), so I add `msp_` prefix (master SPI peripheral) for helper functions of master SPI mode.